source/upgrade.txt

```diff
@@ -35,6 +38,16 @@ Before you upgrade, perform the following actions:
    application when upgrading driver versions by using the
    :ref:`{+stable-api+} <nodejs-stable-api>`.
 
+To upgrade your driver version, run the following command in your application's 
+directory:
+
+.. code-block:: bash
+   
+    npm install mongodb@{+version+}
+
+To upgrade to a different version of the driver, replace ``{+version+}`` with 
```

Comment:
Suggestion:

Rather than depending on the user to identify a specific "version", I think it could be easier to follow by explaining the placement of the information.

E.g.
To upgrade to a different version of the driver, replace the information after the "@" symbol with your preferred version number.

It could also be helpful to link to the official npm documentation as well. E.g.

```
To learn more about the npm install command, see the `npm-install <https://docs.npmjs.com/cli/v10/commands/npm-install?v=true>`__  npm documentation.
```